src/query/mod.rs

```diff
@@ -91,27 +91,23 @@ impl Query {
         let mut config = SessionConfig::default()
             .with_parquet_pruning(true)
             .with_prefer_existing_sort(true)
-            .with_round_robin_repartition(true);
+            .with_information_schema(true)
+            .with_batch_size(1000000)
+            .with_coalesce_batches(true);
```

Comment:
Please add comments on why are we choosing these values and what do they mean for the query